multicluster/test/e2e/service_test.go

```diff
@@ -33,6 +33,13 @@ import (
 	e2euttils "antrea.io/antrea/test/e2e/utils"
 )
 
+const (
```

Comment:
what are these new constants? I'm really confused because they seem unused and completely unrelated to the contents of this file